c/httpserver.c

```diff
@@ -5514,18 +5348,14 @@ static void doHttpReadWork(HttpConversation *conversation, int readBufferSize){
 
   int bytesRead = socketRead(socket,readBuffer,readBufferSize,&returnCode,&reasonCode);
   if (bytesRead < 1) {
-#ifdef DEBUG
-    printf("HTTP desiredBytes = %d bytesRead=%d, so a problem or no more available rc=0x%x, reason=0x%x. conversation=0x%X\n",readBufferSize,bytesRead,returnCode,reasonCode,conversation);
-    fflush(stdout);
-#endif
+    zowelog(NULL, LOG_COMP_HTTPSERVER, ZOWE_LOG_DEBUG3, "HTTP desiredBytes = %d bytesRead=%d, so a problem or no more available rc=0x%x, reason=0x%x. conversation=0x%X\n",
```

Comment:
should be debug1 because this is an interesting error

Reply:
Resolved